### DISCUSSION TITLE
[CUSA15717] Far Cry 6 1.09

### DISCUSSION THREAD

**Discussion by UserGithube** [GoldHEN/GoldHEN_Cheat_Repository#47] [Cheat requests]
Hello everyone. I can't find cheats for Far Cry 6 in 1.09 (CUSA15717). 

Can you help me please. Thanks a lot

**Comment by kmeps4** [comment]
https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42#discussion-4222094
